### PR TITLE
Run all infra on AWS

### PR DESCRIPTION
**Pull request by corbt**
We've successfully migrated all our infra from Render to AWS. This PR contains the infra definition files, as well as a Github Action to deploy whenever we push to `main`.

### CODE REVIEW

File: infra/src/app-env.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ export const environment = new kubernetes.core.v1.Secret(
   nm("app"),
   {
     stringData: {
-      // DATABASE_URL: dbConnectionString,
-      DATABASE_URL: getSecret("DATABASE_URL"),
+      DATABASE_URL: dbConnectionString,
+      // DATABASE_URL: getSecret("DATABASE_URL"),
```
Comment (arcticfly) [inline infra/src/app-env.ts:16]: Do we still need this line?

File: infra/scripts/migrate-prod-db.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import "dotenv/config";
```
Comment (arcticfly) [inline infra/scripts/migrate-prod-db.ts:4]: Nit: Add comment that this file is for documentation purposes.

**Review by arcticfly — APPROVED** [review]
👍

File: infra/src/app-env.ts (hunk 1 above)
Comment (corbt) [reply infra/src/app-env.ts:16]: One primary reason we'd be touching this code would be if we need to migrate the database again, so I'd like to leave this in as a reminder of how we did that last time.

**Pull request merged**